docs/src/pages/addons/writing-addons/index.md

```diff
@@ -107,33 +107,27 @@ Then add the following code to the register.js.
 ```js
 import React from 'react';
 import addons from '@storybook/addons';
+import styled from 'emotion';
 
-const styles = {
-  notesPanel: {
-    margin: 10,
-    fontFamily: 'Arial',
-    fontSize: 14,
-    color: '#444',
-    width: '100%',
-    overflow: 'auto',
-  }
-};
+const NotesPanel = styled('div')({
+  margin: 10,
+  width: '100%',
+  overflow: 'auto',
+});
 
 class Notes extends React.Component {
```

Comment:
I think we should add `propTypes` to the example ;)

Reply:
Also, do you want to mention themes in this doc?

Reply:
I'll mention styling addon components and `lib/components` in this doc.

Do you want to mention/change the section about this:

Reply:
Yeah, let's do this as a separate PR, I'll make a note to do it.

Reply:
👍